src/vstest.console/CommandLine/GenerateFakesUtilities.cs

```diff
@@ -14,7 +14,7 @@ public static class GenerateFakesUtilities
         internal static void GenerateFakesSettings(CommandLineOptions commandLineOptions, IEnumerable<string> sources, ref string runSettingsXml)
         {
             // dynamically compute the fakes datacollector settings
-            if (!commandLineOptions.DisableAutoFakes)
+            if (commandLineOptions == null || !commandLineOptions.DisableAutoFakes)
```

Comment:
>commandLineOptions [](start = 16, length = 18)

Can we not check for design mode instead? Would make things a little more explicit... 
At the least, I think we would want a comment.

Reply:
I'm not sure what you mean here. The change is to enable this to run even in the design mode, so we shouldn't be checking for it

Reply:
So when you said `commandLineOptions == null`, I'm assuming you are checking for designMode, so you can update Fakes setting then too, but that isn't explicit at all here...

Reply:
Oh the design mode check is completely gone, this is just a sanity null check since we check whether `commandLineOptions.DisableAutoFakes` is false/true.